applicationinsights-react-native/src/ReactNativeManualDevicePlugin.ts

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import dynamicProto from "@microsoft/dynamicproto-js";
```

Comment:
We don't really want to cherry-pick this functionality back to the older release -- we just want to move the baseline to support 3.0.9 of AI. (ie. no new functionality)